images/clang/README.md

```diff
@@ -26,4 +26,39 @@ docker pull cgr.dev/chainguard/clang:latest
 <!--getting:end-->
 
 <!--body:start-->
-<!--body:end-->
+
+
+## Use It!
+
+To illustrate working with the Clang Chainguard Image, this section outlines how you can use it to compile a "Hello World!" program written in C. 
+
+To begin, run the following command to create a file named `hello.c` to hold the C code.
+
+```shell
+cat > /tmp/hello.c <<EOF
+#include <stdio.h>
+
+int main() {
+    printf("Hello World!\n");
+    return 0;
+}
+EOF
+```
+
+To simplify cleanup, this command places the file in the `/tmp` temporary directory. 
+
+Next, run the following `docker` command. This will mount the contents of your local `/tmp` directory (including the `hello.c` file) into the container's `work` directory. Once there, Clang will compile the C code into an executable program named `hello`.
+
+```shell
+docker run --rm -v /tmp:/work cgr.dev/chainguard/clang:latest hello.c -o hello
+```
+
+The `hello` program will be stored back in your local `/tmp` directory. You can test that everything worked correctly by executing this program.
+
+```shell
+/tmp/hello
```

Comment:
My comment got deleted :scream:

I had trouble with this part. I tried on Linux web terminal and my local macos

**Linux**:

```sh
a0def3df:~$ ./tmp/hello
bash: ./tmp/hello: No such file or directory
a0def3df:~$ cd /tmp
a0def3df:/tmp$ ./hello
bash: ./hello: cannot execute: required file not found
a0def3df:/tmp$ ls
hello    hello.c
```

**Mac**:

```sh
Chaingourd:tmp ltagliaferri$ /tmp/hello
-bash: /tmp/hello: cannot execute binary file: Exec format error
Chaingourd:tmp ltagliaferri$ ./hello
-bash: ./hello: cannot execute binary file: Exec format error
```

Also tried variations:

```sh
./hello.out
./hello
```

The Docker image is clearly working as it creates the `hello` file.